web/src/components/shared/navbar.tsx

```diff
@@ -50,10 +54,17 @@ const StyledLink = styled(LinksSmall)`
   }
 `;
 
-const NavBar = () => {
+const NavBar = ({ categories }: { categories: CategoryConfiguration[] }) => {
```

Comment:
I didn't realize before but the default behavior for a navbar in mobile is to be floating, hide when you scroll down and appear again when you scroll up. For that, we can use something like ReactHeadroom. Check the navbar implementation of my own [website](https://github.com/Raagh/patferraggi.dev/blob/master/src/components/shared/navbar.js)